### PR TITLE
LPD-28249 Avoid trying to delete system object relationships

### PR DESCRIPTION
**Pull request by igor-franca**
Related [Ticket](https://liferay.atlassian.net/browse/LPD-33330)

### CODE REVIEW

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro [hunk 1]
```diff
@@ -930,29 +930,29 @@ definition {
...
-	}
-
-	@summary = "Default summary"
-	macro deleteAllRelationships() {
+	macro deleteAllCustomObjectRelationships() {
 		var objectDefinitionIdList = JSONObject.getAllObjectDefinitionsId();
 
 		for (var objectDefinitionId : list ${objectDefinitionIdList}) {
-			var relationshipIdList = JSONObject.getObjectRelationships(objectDefinitionId = ${objectDefinitionId});
+			var customObjectRelationshipIdList = JSONObject.getCustomObjectRelationships(objectDefinitionId = ${objectDefinitionId});
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro:937]: @igor-franca rename to `customObjectRelationshipIds`

File: modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro [hunk 2]
```diff
@@ -1133,6 +1133,32 @@ definition {
 		return ${objectDefinitionIdList};
 	}
 
+	@summary = "Default summary"
+	macro getCustomObjectRelationships(objectName = null) {
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro:1137]: @igor-franca rename to `getCustomObjectRelationshipIds`

File: modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro [hunk 3]
```diff
@@ -1133,6 +1133,32 @@ definition {
...
+		}
+
+		var curl = '''
+			${portalURL}/o/object-admin/v1.0/object-definitions/${objectDefinitionId}/object-relationships?filter=system%20eq%20false \
+				-u ${userEmailAddress}:${userPassword}
+		''';
+
+		var customObjectRelationshipIdList = JSONCurlUtil.get(${curl}, "$.items..id");
+
+		return ${customObjectRelationshipIdList};
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro:1159]: rename to `customObjectRelationshipIds`

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]
Good job @igor-franca!!

see my comments!

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:test:object

**Comment by igor-franca** [comment]
Thanks for the review @MarinhoFeliphe , see my changes [here](https://github.com/liferay-objects/liferay-portal/compare/e669aebb19f8b914ae3e792a140c0740d472538c..953f0de8b115001a4766be4b5f723da79e19b25f).

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:report:57321944

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

good job, @igor-franca!

**Comment by igor-franca** [comment]
I've seen some unique failures that can be flaky. I will run the CI again

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:forward

**Pull request closed without merging**